src/ray/raylet/node_manager.cc

```diff
@@ -1608,6 +1606,8 @@ void NodeManager::DisconnectClient(const std::shared_ptr<ClientConnection> &clie
     }
   }
 
+  local_task_manager_->ClearWorkerBacklog(worker->WorkerId());
```

Comment:
can you briefly tell me what was the exact bug? Is it because we didn't clear backlogs from driver?

Reply:
Right, updated the PR description